### PR TITLE
API1800 support for Hypervisor Cluster Profile

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
API1800 support for HCP

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [x] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Review by VenkateshRavula — APPROVED** [review]

File: spec/unit/provider/oneview_hypervisor_cluster_profile_c7000_spec.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@
 require 'spec_helper'
 
 provider_class = Puppet::Type.type(:oneview_hypervisor_cluster_profile).provider(:c7000)
-api_version = 800
+api_version = 1800
```
Comment (chebroluharika) [inline spec/unit/provider/oneview_hypervisor_cluster_profile_c7000_spec.rb:20]: is there any reason for changing this api version here?

Comment (yuvirani) [reply spec/unit/provider/oneview_hypervisor_cluster_profile_c7000_spec.rb:20]: Added UT for soft delete.

Comment (chebroluharika) [reply spec/unit/provider/oneview_hypervisor_cluster_profile_c7000_spec.rb:20]: add the same for synergy spec also

Comment (yuvirani) [reply spec/unit/provider/oneview_hypervisor_cluster_profile_c7000_spec.rb:20]: Done

**Review by chebroluharika — APPROVED** [review]

**Pull request merged**